### PR TITLE
Feature/repo clean up

### PR DESCRIPTION
**Pull request by sethrosetter**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- Slight restructure of mobile-testkit per walkthough the other day
- Removed unimplemented stale directory from robot days. I filed tracking tickets for each scenario that has not yet been implemented
  - https://github.com/couchbaselabs/mobile-testkit/issues/1353
  - https://github.com/couchbaselabs/mobile-testkit/issues/1354
  - https://github.com/couchbaselabs/mobile-testkit/issues/1355
  - https://github.com/couchbaselabs/mobile-testkit/issues/1356

### CODE REVIEW

File: testsuites/Tags.md [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-### Feature
-    - replication
-    - changes
-    - views
-    - onlineoffline
-    - roles
-    - sync
-    - ttl
-    - oidc
-    - compression
```
Comment (sridevi-15) [inline testsuites/Tags.md]: test review

Comment (sethrosetter) [inline testsuites/Tags.md]: test response

File: utilities/build_and_deploy_liteserv.sh [hunk 2]
```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env bash
```
Comment (sridevi-15) [inline utilities/build_and_deploy_liteserv.sh]: Shall we have this script for customized build?

Comment (sethrosetter) [inline utilities/build_and_deploy_liteserv.sh]: Readded the script

**Comment by sethrosetter** [comment]
I added instructions to each github ticket on how to find the original robot files.

**Comment by raghusarangapani** [comment]
Looks good to me

**Pull request merged**